src/components/Modal/Modal.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { closeModal } from './../../actions/index';
+
+import { ReactComponent as Close } from './../../assets/Close.svg';
```

Comment:
```suggestion
import { ReactComponent as ButtonClose } from './../../assets/button-close.svg';
```

Reply:
Capital letter at the beginning of a file should be left for components not svgs, lets give this a better name too